package.json

```diff
@@ -17,9 +17,11 @@
   "author": "Paulus Schoutsen <Paulus@PaulusSchoutsen.nl> (http://paulusschoutsen.nl)",
   "license": "Apache-2.0",
   "dependencies": {
-    "@material/mwc-base": "^0.6.0",
-    "@material/mwc-button": "^0.6.0",
-    "@material/mwc-ripple": "^0.6.0",
+    "@material/mwc-base": "^0.8.0",
```

Comment:
For next time, making this a new PR that could have been merged independently would have helped with unblocking Thomas' PR.